tests/test_segmentation.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+import bpy
+import os
+import cv2
+
+from src.controllers.segmentation import Segmentation
+
+class SegmentationTest(unittest.TestCase):
+    """
+    This class contains unit tests for the Segmentation class.
+    """
+
+    def setUp(self):
+        """
+        This method is called before each test method.
+        It sets up the output file name for the test.
+        """
+        self.output_file = "test_segmentation.png"
+
+    def tearDown(self):
+        """
+        This method is called after each test method.
+        It removes the output file created during the test.
+        """
+        os.remove(self.output_file)
+
+    def test(self):
+        """
+        This method tests the segmentation of a cube object.
+        It asserts that the central point of the image is white and the background is black.
+        """
+        collection = bpy.data.collections.get("Collection")
+        cube = collection.objects.get("Cube")
+        collection.objects.unlink(cube)
+        bpy.ops.mesh.primitive_cube_add()
+        o = bpy.context.active_object
```

Comment:
Only note would be to use meaningful variable names instead of 'o'

Reply:
Sure.